examples/map-cat/Dockerfile

```diff
@@ -1,16 +1,16 @@
+# use a minimal rust base image
 FROM rust:1.75-bookworm as build
 
 RUN apt-get update
 RUN apt-get install protobuf-compiler -y
 
 # create a new empty shell project
-WORKDIR /examples
+WORKDIR /numaflow-rs
 
-# copy your source tree
-COPY src ./src
```

Comment:
we might copy unwanted files, right?

Reply:
I guess rather than copy the entire root directory with `COPY ./ ./,` the only necessary things necessary to copy over would be src/, proto/, Cargo.toml, and build.rs? Is there anything else?

Reply:
for examples, we do not require proto, build.rs, etc., right?